### PR TITLE
Support unacked writes (mgo)

### PR DESCRIPTION
**Pull request by TomerYakir**

### CODE REVIEW

File: proxy.go [hunk 1]
```diff
@@ -204,10 +204,11 @@ func (ps *ProxySession) doLoop(pooledConn *PooledConnection) (*PooledConnection,
 	}
 
 	var respInter ResponseInterceptor
+	expectingResponse := true
 	if ps.interceptor != nil {
 		ps.interceptor.TrackRequest(m.Header())
 
-		m, respInter, err = ps.interceptor.InterceptClientToMongo(m)
+		m, respInter, expectingResponse, err = ps.interceptor.InterceptClientToMongo(m)
```
Comment (tolsen) [inline proxy.go]: same comment as other PR.  does the if on line 219 need to modified too?

File: session.go [hunk 2]
```diff
@@ -31,6 +33,10 @@ func (s *Session) Logf(level slogger.Level, messageFmt string, args ...interface
 	return s.logger.Logf(level, messageFmt, args...)
 }
 
+func (s *Session) SetRemoteAddr(v net.Addr) {
```
Comment (tolsen) [inline session.go]: why do we need to add this?

File: proxy.go (hunk 1 above)
Comment (TomerYakir) [reply proxy.go]: https://github.com/erh/mongonet/pull/37#discussion_r384983311

File: session.go (hunk 2 above)
Comment (TomerYakir) [reply session.go]: this was part of that commit - https://github.com/erh/mongonet/pull/39/commits/73ab45ff832fa959e6b6873ceef7c2961f2f944d. I'm not entirely sure why it's showing in this PR.

Comment (tolsen) [reply session.go]: This is a red flag as to something being wrong with the branch.  Could you please investigate?

**Comment by TomerYakir** [comment]
will investigate what's wrong with the branch and open a new PR

**Pull request closed without merging**